animated_sprite.h

```diff
@@ -0,0 +1,94 @@
+#ifndef ANIMATEDSPRITE_H
+#define ANIMATEDSPRITE_H
+
+#include <SDL2/SDL.h>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+#include "sprite.h"
+#include "vector2.h"
+
+class AnimatedSprite : public Sprite {
+ public:
+  // Constructor takes the name of the sprite sheet, the source rectangle for
+  // the first frame, the renderer, and the time between each frame.
+  AnimatedSprite(std::string sprite_sheet_name,
+                 SDL_Rect first_sprite_coord_location, SDL_Renderer* renderer,
+                 float time_to_update_sprite, SDL_Rect destination);
```

Comment:
clarify in the comment and in the variable name what units are being use for time.

```
// and the time in milliseconds between each frame.

float update_time_ms
```

Reply:
fix